### PR TITLE
[SELC - 4989] feat: added sent notification logic

### PR DESCRIPTION
**Pull request by gianmarcoplutino**
#### List of Changes

- added sent notification logic when status is COMPLETED or DELETED

#### Motivation and Context

the function that allows sending to queue has been invoked. This is only done when the status is COMPLETED or DELETED

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/onboarding-cdc/src/main/java/it/pagopa/selfcare/onboarding/event/mapper/OnboardingMapper.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public interface OnboardingMapper {
+    @Mapping(target = "createdAt", source = "createdAt", qualifiedByName = "toOffsetDateTime")
+    @Mapping(target = "activatedAt", source = "activatedAt", qualifiedByName = "toOffsetDateTime")
+    @Mapping(target = "expiringDate", source = "expiringDate", qualifiedByName = "toOffsetDateTime")
+    @Mapping(target = "updatedAt", source = "updatedAt", qualifiedByName = "toOffsetDateTime")
+    Onboarding toEntity(it.pagopa.selfcare.onboarding.event.entity.Onboarding model);
+    @Named("toOffsetDateTime")
+    default OffsetDateTime map(String value) {
+        LocalDateTime localDateTime = LocalDateTime.parse(value);
+        ZonedDateTime zonedDateTime = localDateTime.atZone(ZoneId.of("Europe/London"));
```
Comment (manuraf) [inline apps/onboarding-cdc/src/main/java/it/pagopa/selfcare/onboarding/event/mapper/OnboardingMapper.java:25]: why London?

Comment (gianmarcoplutino) [reply apps/onboarding-cdc/src/main/java/it/pagopa/selfcare/onboarding/event/mapper/OnboardingMapper.java:25]: I saw that on the DB we save the dates as UTC +0 and searching on the internet it should be London, I don't actually know if my reasoning is correct

Comment (gianmarcoplutino) [reply apps/onboarding-cdc/src/main/java/it/pagopa/selfcare/onboarding/event/mapper/OnboardingMapper.java]: Changed to default

File: apps/onboarding-cdc/src/main/resources/application.properties [hunk 2]
```diff
@@ -14,4 +14,10 @@ onboarding-cdc.storage.connection-string=${STORAGE_CONNECTION_STRING:UseDevelopm
 
 onboarding-cdc.retry.min-backoff=${ONBOARDING-CDC-RETRY-MIN-BACKOFF:10}
 onboarding-cdc.retry.max-backoff=${ONBOARDING-CDC-RETRY-MAX-BACKOFF:12}
```
Comment (manuraf) [inline apps/onboarding-cdc/src/main/resources/application.properties:16]: maybe 12 hours it too long....you could consider to reduce at most 1 hours

Comment (gianmarcoplutino) [reply apps/onboarding-cdc/src/main/resources/application.properties:16]: fix it

**Review by andrea-putzu — APPROVED** [review]

**Pull request merged**